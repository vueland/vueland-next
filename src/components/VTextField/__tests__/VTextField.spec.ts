import { mount, VueWrapper } from '@vue/test-utils'
import { VTextField } from '../index'

describe('VtextField', () => {
  let mountFunction: (options?: any) => VueWrapper<any>

  beforeEach(() => {
    mountFunction = (options = {}) => mount(VTextField, {
      ...options,
      global: {
        provide: {
          $options: null,
        },
      },
    })
  })

  it('should mount component and match snapshot', () => {
    const cmp = mountFunction()

    expect(cmp.html()).toMatchSnapshot()
  })
})
