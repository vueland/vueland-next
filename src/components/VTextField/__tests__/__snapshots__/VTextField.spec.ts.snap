// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VtextField should mount component and match snapshot 1`] = `
<div class="v-input">
  <div class="v-input__slot primary--text">
    <div class="v-input__field-slot">
      <label class="v-label v-label--on-field primary--text"
             style="left: 0px; position: absolute;"
      >
      </label>
      <div class="v-text-field">
        <input class="v-text-field__input"
               type="text"
               style="color: rgb(33, 33, 33); caret-color: #212121;"
        >
      </div>
    </div>
    <div class="v-input__status">
      <transition-stub>
      </transition-stub>
    </div>
  </div>
</div>
`;
